content/css/concepts/filter-functions/terms/opacity/opacity.md

```diff
@@ -38,3 +38,14 @@ Set the opacity of our element to `50%`:
   filter: opacity(50%);
 }
 ```
+
+## Example 2
+
+Set the opacity of our element to `30%`:
+
+```css
+.banner-image {
+  filter: opacity(30%);
+}
+```
+![css-filterfunction-opacity-example2](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunction-opacity-example2.png)
```

Comment:
@giliopoulosaz in the generated issue you were instructed to add media of the current example. No need to create a new example and add media for it.